app/views/equipment_models/index.html.erb

```diff
@@ -5,27 +5,29 @@
 <hr>
 <% if can? :manage, EquipmentModel %>
   <% if @category %>
-    <%= link_to "New #{@category.name.singularize}", new_category_equipment_model_path(@category), :class => 'btn btn-primary' %>
+    <%= link_to "New #{@category.name.singularize}", new_category_equipment_model_path(@category), class:  'btn btn-primary' %>
     <%= link_to "#{@category.name} Reservations Calendar", calendar_category_path(@category), class: 'btn btn-default' %>
-    <%= link_to "View All Equipment Models", equipment_models_path, :class => 'btn btn-default' %>
+    <%= link_to "View All Equipment Models", equipment_models_path, class:  'btn btn-default' %>
+    <%= link_to "Import Equipment Models", equip_import_page_path, class:  "btn btn-default" %>
+    <%= link_to "Export Equipment Data", equipment_models_path(format: 'zip'), class:  "btn btn-default" %>
```

Comment:
Should we make this the `category_path(@category, format: 'zip')` instead since it's a scoped view and you built scoped export?